### PR TITLE
fix #5656: Fixed the position of the button on Notification when displaying multiple lines of text

### LINKED ISSUE #5656: [ui/ux][notification]Buttons are strange when displaying multiple lines of text
### Description
Current

Expected

### PR DESCRIPTION
**Pull request by BroKun**
This is a long-standing problem, it seems that removing the useless style settings is fine.

Signed-off-by: Brokun <brokun0128@gmail.com>

### CODE REVIEW

File: packages/messages/src/browser/style/notifications.css [hunk 1]
```diff
@@ -15,188 +15,188 @@
  ********************************************************************************/
 
 .theia-NotificationsContainer {
-    pointer-events: none;
```
Comment (akosyakov) [inline packages/messages/src/browser/style/notifications.css]: @BroKun Could you elaborate please what really was changed? It would be good to separate a refactoring change in a separate commit to simplify a review. We are going to include this rule in the review guideline: https://github.com/theia-ide/theia/pull/5616#issuecomment-508428861

File: packages/messages/src/browser/style/notifications.css [hunk 2]
```diff
@@ -15,188 +15,188 @@
...
-    }
-    to {
-        opacity: 1;
-        -webkit-transform: none;
-        transform: none;
-    }
+  from {
+    -webkit-transform: translate3d(0, -100%, 0);
+            transform: translate3d(0, -100%, 0);
+    opacity: 0;
```
Comment (AlexTugarev) [inline packages/messages/src/browser/style/notifications.css]: @BroKun thanks for the PR!

I agree with @akosyakov, even with ignoring whitespace changes it takes some time to spot the real changes.

**Comment by BroKun** [comment]
Sorry, there are some automatic formatting in it, I will deal with it.

**Comment by BroKun** [comment]
> Sorry, there are some automatic formatting in it, I will deal with it.

It’s clean now @akosyakov @AlexTugarev

**Comment by vince-fugnitto** [comment]
It works fine for me, I tested a message with multiple lines :)
Do you think the commit message should be updated, I don't believe it addresses **button performance**.

**Comment by BroKun** [comment]
> It works fine for me, I tested a message with multiple lines :)
> Do you think the commit message should be updated, I don't believe it addresses **button performance**.

Forgive me for my poor English.

**Comment by vince-fugnitto** [comment]
> > It works fine for me, I tested a message with multiple lines :)
> > Do you think the commit message should be updated, I don't believe it addresses **button performance**.
> 
> Forgive me for my poor English.

No that's alright, don't worry! :)

**Comment by vince-fugnitto** [comment]
The PR is fine for the moment, but I still think we should try and update our notifications (or messages) similarly to vscode. They're easier to read, take up less space and look nicer in general.

<div align='center'>

image


</div>

File: packages/messages/src/browser/style/notifications.css (hunk 2 above)
Comment (AlexTugarev) [reply packages/messages/src/browser/style/notifications.css]: thanks for the cleanup!
